proto/streamlit/proto/Markdown.proto

```diff
@@ -23,4 +23,15 @@ message Markdown {
 
   bool allow_html = 2;
   bool is_caption = 3;
```

Comment:
After adding the `element_type` field, `is_caption` becomes redundant and obsolete , my plan is to remove the `is_caption` flag after careful review of the frontend code for st.caption and researching how to do that without breaking backward capability for this proto file.